### ISSUE TITLE
cTiVo asking for Contacts permission in Sierra

### ISSUE THREAD

**Issue by mackworth** [dscottbuch/cTiVo#203]
tl;dr: Just say No, and everything should be fine.

Moving from issue #201:
@patmacster: I do have a question why did ctivo want access to my contacts?

@mackworth: Nope; that's very weird. cTiVo absolutely shouldn't be asking for contacts. What did that look like? Go to Security & Privacy > Privacy tab > Contacts > and if it's there, disable ctivo access.

@wkoffel: Also @mackworth , I do see that cTivo asked for access to my contacts (this is on macOS sierra), for what it's worth.

@mackworth: Just saw the contacts note. I'll look into that

@mackworth: The contacts thing is baffling me. With two reports, I gotta assume it's real :) .
But we don't even load the framework that accesses contacts, and the docs say that starting in 10.10, if we do access the contacts, then we have to provide a string explaining why to the user, or the program will crash, and we don't provide that string.
Questions:
When did it ask for permission? On initial launch, or doing some other operation?
Do you remember what it says?
In your System Preferences > Security & Privacy > Privacy tab > Contacts, does cTiVo appear?
If not, what apps do appear?

@wkoffel: It prompted me to use Contacts on initial launch. And yes, it shows in the privacy preferences: https://cl.ly/3n0L0e0F2X3d
Also, notable, when I tried the new beta that you suggested, I believe it prompted me again for that version, even though it already had permissions.
This is on macOS 10.12 Sierra. I know that Apple is constantly tweaking the sandboxing and permissions, so it could be a side effect of the new OS. Just thinking that some call that seems innocuous and unrelated could now require Contacts access (something like getting the user's name, or something about access to the keychain? Now I'm just totally guessing, but you get the gist....)

**Comment by Ramias1** [comment]
Did it to me just now too.

**Comment by mackworth** [comment]
I still haven't been able to track this down. On the other hand, just saying "no" seems to have no ill effects whatsoever.

**Comment by mackworth** [comment]
Fixed in beta3
